magenta/music/musicxml_parser_test.py

```diff
@@ -0,0 +1,190 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test to ensure correct import of MusicXML."""
+
+from collections import defaultdict
+import os.path
+
+# internal imports
+import tensorflow as tf
+from magenta.music.musicxml_parser import MusicXMLDocument
+from magenta.music.musicxml_reader import musicxml_to_sequence_proto
+
+class MusicXMLParserTest(tf.test.TestCase):
```

Comment:
I initially created the test cases based on what the MIDI test cases did at that time and added one for the compressed MusicXML file. What other test cases do you think would be useful?